data/tlg0057/tlg086/tlg0057.tlg086.1st1K-grc1.xml

```diff
@@ -0,0 +1,537 @@
+                    <persName>Gregory R. Crane, Universität Leipzig</persName>
+                    <persName>Uwe Vagelpohl, University of Warwick</persName>
+                </respStmt>
+                <respStmt>
+                    <resp>Keyboarding</resp>
+                    <orgName xml:id="DDD">Digital Divide Data</orgName>
+                </respStmt>
+                <respStmt>
+                    <resp>Editor-in-Chief, Perseus Digital Library</resp>
+                    <persName>Gregory R. Crane</persName>
+                </respStmt>
+                <respStmt>
+                    <resp>Project Manager (University of Leipzig), 2016 - present</resp>
+                    <persName>Matt Munson</persName>
+                </respStmt>
+                <respStmt>
+                    <resp>Project Assistant (University of Leipzig), 2015 - present</resp>
+                    <persName>Annette Geßner</persName>
+                </respStmt>
+                <respStmt>
+                    <resp>Lead Developer (University of Leipzig)</resp>
+                    <persName>Thibault Clérice</persName>
+                </respStmt>
+                <respStmt>
+                    <resp>Technical Advisor</resp>
+                    <persName>Bruce Robertson</persName>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>A Digital Corpus for Graeco-Arabic Studies, funded by the Andrew W.
+                    Mellon Foundation</authority>
+                <publisher>University of Leipzig</publisher>
+                <pubPlace>Germany</pubPlace>
+                <idno type="filename">tlg0057.tlg086.1st1K-grc1.xml</idno>
+                <availability>
+                    <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available
+                        under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</licence>
+                </availability>
+                <date>2016</date>
+            </publicationStmt>
+            <sourceDesc>
+                <biblStruct>
+                    <monogr>
+                        <title xml:lang="lat">Claudii Galeni Opera Omnia</title>
+                        <author>Galen</author>
+                        <editor>Karl Gottlob Kühn</editor>
+                        <imprint>
+                            <publisher>Cnobloch</publisher>
+                            <pubPlace xml:lang="lat">Lipsiae</pubPlace>
+                            <biblScope unit="vol">15</biblScope>
+                            <biblScope unit="pp" from="174" to="223">174-223</biblScope>
+                            <date>1822</date>
+                        </imprint>
+                    </monogr>
+                    <ref target="https://www.graeco-arabic-studies.org/fileadmin/user_upload/texts.xml/Galen-Gr_098.xml">A Digital Corpus for Graeco-Arabic Studies</ref>
+                </biblStruct>
```

Comment:
You should also add a `<ref>` element pointing to the scan on Hathi Trust as well. That is here: https://babel.hathitrust.org/cgi/pt?id=coo.31924008819116;view=1up;seq=143